app/views/components/blockEditor.html

```diff
@@ -47,7 +47,7 @@
         </div>
 
         <div class="form-group" ng-if="block.type === 'paragraphContent'">
-          <textarea ng-model="block.content" class="form-control" style="height:400px;"></textarea>
+          <textarea ng-model="block.content" class="form-control" style="height:400px;" tooltip="{{'Supports hyperlink html format.' | translate}}"></textarea>
```

Comment:
I think this should have the same content as the comment I made for the titles. I also would like to make this show on hovering over a question mark but idk where to place it... Maybe it is fine hovering over the whole input but that means it is showing the entire time your mouse is there.

Maybe we could add a Description heading and put the question mark next to it just link the title. Or maybe for this block only, combine the tooltips of the title and description fields to be shown next to the "Title" text and say something like "Title and description support HTML links Example..."

What do you guys think? @jayakrishnanv @hlbraddock @jcsaplar